component/src/main/java/org/wso2/rule/validator/validator/JsonRulesetValidator.java

```diff
@@ -28,7 +29,12 @@
  * Ruleset validation for JSON files
  */
 public class JsonRulesetValidator extends RulesetValidator {
-    public static List<RulesetValidationError> validateRuleset(String rulesetString) {
-        return RulesetValidator.validate((Map<String, Object>) JsonPath.parse(rulesetString).json());
+    public static List<RulesetValidationError> validateRuleset(String rulesetString)
+            throws InvalidContentTypeException {
+        Object jsonContent = JsonPath.parse(rulesetString).json();
+        if (!(jsonContent instanceof Map)) {
+            throw new InvalidContentTypeException("Invalid JSON content.");
```

Comment:
Shall we change this to `Invalid JSON Ruleset content`? Because technically just because it's not a Map does not mean it's not valid JSON. It could be valid JSON but not a valid ruleset since rulesets are Maps.